regolith/helpers/u_contacthelper.py

```diff
@@ -0,0 +1,131 @@
+"""Helper for adding a new person to the contacts collection.
+
+"""
+import datetime as dt
+from nameparser import HumanName
+import dateutil.parser as date_parser
+import sys
+import uuid
+
+from regolith.helpers.basehelper import DbHelperBase
+from regolith.fsclient import _id_key
+from regolith.tools import all_docs_from_collection
+
+
+TARGET_COLL = "contacts"
+
+def subparser(subpi):
+    subpi.add_argument("name", help="first name space last name in quotes")
+    subpi.add_argument("institution", help="person's institution.  Can be inf"
+                                            "short form such as columbiau and will "
+                                            "be retrieved from institutions collection")
+    # subpi.add_argument("-e", "--email",
+    #                    help="email address")
```

Comment:
Hi professor, @sbillinge
I could not add a field email, because it does not pass validators.py (ensure_email). 
Can you give me an idea of what I could do?

Reply:
an email field shouldn't be added to the db unless an email is specified at the command line, and then it mus be a valid email. ``something@thing.else`` so things should work fine.

Reply:
when I have a valid email (in this case, mine), it gives me the following error:
```
email = 'dy2403@barnard.edu-a'

    def ensure_email(email):
        """Ensures the email top-level key is well formed."""
>       email["url"] = ensure_string(email["url"])
E       TypeError: string indices must be integers
```

Reply:
added it as an issue in regolith (as we talked in the call).